pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -1386,6 +1397,18 @@ public void testDeleteTenant() throws Exception {
         assertFalse(pulsar.getLocalMetadataStore().exists(bundleDataPath).join());
     }
 
+    private TreeSet<String> queryTopicsByNamespace(String namespace){
+        NamespaceName namespaceName = NamespaceName.get(namespace);
+        TreeSet<String> topics = new TreeSet<>();
+        topics.addAll(pulsar.getNamespaceService()
+                .getListOfPersistentTopics(NamespaceName.get(namespace)).join());
+        topics.addAll(pulsar.getPulsarResources().getNamespaceResources().getPartitionedTopicResources()
```

Comment:
I think query `getListOfPersistentTopics` is enough, because `__change_events` is persistent.

Reply:
And we don't need to check partitioned topics? Because we will eventually create the partitons.

Reply:
Hi @Technoboy- 

> I think query getListOfPersistentTopics is enough, because __change_events is persistent.

already remove the logic that queries non-persistent topic

Hi @codelipenghui 

> And we don't need to check partitioned topics? Because we will eventually create the partitons.

already add the partitioned topics check.